docbook/reference/en/en-US/modules/Installation_Configuration.xml

```diff
@@ -442,9 +442,299 @@ public class MyApplication extends Application
+            <classname>ConfigSource</classname>s are searched in order until a value is found.
+        </para>
+        <para>
+            MicroProfile Config mandates the presence of the following <classname>ConfigSource</classname>s:
+        </para>
+        <orderedlist>
+            <listitem>a <classname>ConfigSource</classname> based on System.getProperties() (ordinal = 400)</listitem>
+            <listitem>a <classname>ConfigSource</classname> based on System.getenv() (ordinal = 300)</listitem>
+            <listitem>a <classname>ConfigSource</classname> for each META-INF/microprofile-config.properties file on the ClassPath,
+            separately configurable via a config_ordinal property inside each file (default ordinal = 100)</listitem>
+        </orderedlist>
+        <para>
+            Note that a property which is found among the System properties and which is also in the System environment will be assigned
+            the System property value because of the relative priorities of the <classname>ConfigSource</classname>s.
+        </para>
+        <para>
+            The set of <classname>ConfigSources</classname> is extensible. For example, smallrye-config
+            (<ulink url="https://github.com/smallrye/smallrye-config">https://github.com/smallrye/smallrye-config</ulink>),
+            the implementation of the MicroProfile Config specification currently used by RESTEasy, adds, among others,
+            the following kinds of <classname>ConfigSource</classname>s:
+        </para>
+        <orderedlist>
+            <listitem><classname>PropertiesConfigSource</classname> creates a <classname>ConfigSource</classname> from a
+            Java <classname>Properties</classname> object or a Map&lt;String, String&gt; object or a properties file
+            (referenced by its URL) (default ordinal = 100).</listitem>
+            <listitem><classname>DirConfigSource</classname> creates a <classname>ConfigSource</classname> that will look into a directory
+            where each file corresponds to a property (the file name is the property key and its textual content is the property value).
+            This <classname>ConfigSource</classname> can be used to read configuration from Kubernetes ConfigMap (default ordinal = 100).</listitem>
+        </orderedlist>
+        <para>
+            These can be registered programmatically by using an instance of <classname>ConfigProviderResolver</classname>:
+        </para>
+        <programlisting>
+Config config = new PropertiesConfigSource("file:/// ...");
+ConfigProviderResolver.instance().registerConfig(config, getClass().getClassLoader());
+        </programlisting>
+        <para>where <classname>ConfigProviderResolver</classname> is part of the Eclipse MicroProfile Config API.</para>
+        <para>
+            If the application is running in Wildfly, then Wildfly provides another set of <classname>ConfigSource</classname>s,

```

Comment:
s/Wildfly/WildFly/

Reply:
@asoldano, wow. I don't think I ever noticed that!! ;-)

Reply:
Done